Userland/Libraries/LibGfx/ImageFormats/JPEGXLLoader.cpp

```diff
@@ -399,6 +399,22 @@ static ErrorOr<ToneMapping> read_tone_mapping(LittleEndianInputBitStream& stream
 }
 
 struct OpsinInverseMatrix {
+    f32 inv_mat00 = 11.031566901960783;
```

Comment:
Do these number come from the spec? Can we put a spec ref above them?

Reply:
I'm not too worried about finding stuff in the spec as everything has quite descriptive name. I have another PR touching this area, so I'll add it in there.